setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
+ </tr>
+</table>
+
+Using an IDE is optional. However, we recommend using an IntelliJ IDE with Flutter for code completion, 
+inline error checking, and visual debugging features.
+{: .alert-warning}
 
 ## System requirements
 
 To install and run Flutter, your development environment must meet these minimum requirements:
 
 * **Operating Systems**: Mac or Linux (64-bit). Windows support is [in progress](https://github.com/flutter/flutter/issues/138).
+* **Disk Space**: 710 MB on Mac, 558 MB on Linux (does not include disk space for Xcode or Android Studio).
 * **Tools**: Flutter depends on these command-line tools being available on your environment.
   * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
 
 ## Get the Flutter SDK
 
-To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+1. To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
 
-<pre>
-$ git clone https://github.com/flutter/flutter.git
-$ export PATH=`pwd`/flutter/bin:$PATH
-</pre>
+   ```
+   $ git clone https://github.com/flutter/flutter.git
+   $ export PATH=`pwd`/flutter/bin:$PATH
+   ```
 
-Run the following command to see if there are any dependencies you need to install to complete
+   where `pwd` is the current working directory, for example:
+   ```
+   $ export PATH=/Users/obiwan/flutter/bin:$PATH
+   ```
+
+   The above command sets your PATH variable temporarily, for the current terminal window.
+   The steps for modifying this variable permanently for all terminal sessions are machine-
+   specific. Typically you add a line to a file that is executed whenever you open 
+   a new window:
+
+   ```
+   export PATH=~/flutter/bin
+   ```
+and then run the `source <filename>` command to refresh the window. For example on Linux, 
+you might edit and source `~/.bash_profile`. On macOS, it might be `/etc/bashrc`.
+
+1. Verify that the `flutter/bin` directory is now in your PATH by running:
+   ```
+   $ echo $PATH
+   ```
+
+1. Run the following command to see if there are any dependencies you need to install to complete
 the setup:
 
-```
-$ flutter doctor
-```
+   ```
+   $ flutter doctor
+   ```
+
+   This command checks your environment and displays a report to the terminal window.
+   The Dart SDK is bundled with Flutter; it is not necessary to install Dart separately.
+   Check the output carefully for other software you may need to install or further 
+   tasks to perform (shown in **bold** text).
+
+   For example:
+   ```
+    ✗ Xcode end user license agreement not signed; open Xcode or run the command 'sudo xcodebuild -license'.
+
+    ✗ Brew not installed; use this to install tools for iOS device development.
+
+
+      Download brew at http://brew.sh/.
+   ```
 
-The first time you run the `flutter` command, it will download its dependencies and compile
+   The first time you run the `flutter` command, it downloads its dependencies and compiles
```

Comment:
Added "own" so it reads "its own". From the context, users should not be confused. If they are confused we'll take another pass.